include/nbl/asset/IAccelerationStructure.h

```diff
@@ -140,6 +140,11 @@ class NBL_API IAccelerationStructure : public IDescriptor
 			E_GEOM_TYPE					type;
 			E_GEOM_FLAGS				flags; // change to core::bitflags later
 			GeometryData<AddressType>	data;
+
+			inline bool operator!=(const Geometry& other) const
+			{
+				return !std::memcmp(this, &other, sizeof(other));
+			}
```

Comment:
wonder if it all would have worked if you didnt mark `IAccelerationStructure` as wholesale `NBL_API` and just the 3 non-inline methods it has